### PR TITLE
Leveling in code review

### PR DESCRIPTION
**Pull request by AndyCoveo**

### CODE REVIEW

File: _posts/2022-02-05-leveling-code-review.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- Should another Subject Matter Expert (SME) be involved in this review?
+
+The best way to get to this level is to start discussing and understanding what other teams, outside of your service, are doing. The more you’ll learn about how they work and their current issue, the better perspective you’ll have on the impact of the changes you are making.
+
+
+# Is this it?
+
+As you have probably noticed, those levels are mostly oriented towards someone working on a micro/macro service within a company. Nonetheless, I hope there can be some value in there even if this isn’t your current situation. One thing you’ll find is missing from this post is how to provide effective and constructive feedback, which will be covered in a future post on this blog, stay tuned :). If you are impatient, you can go and read what Google has to say about it in their [9th chapter of Software Engineering at Google](https://abseil.io/resources/swe_at_google.2.pdf).
+
+_Thanks to Devin Lafrenière, David Lévesque, Maxime Lachapelle, Alexandre Moreau, Kevin Larose, Samuel Begin, Louis Bompart, Martin Ouellet and Marc Sanfaçon who were kind enough to review a text on reviews._
```
Comment (amoreauCoveo) [inline _posts/2022-02-05-leveling-code-review.md:84]: https://i.imgflip.com/49a5w7.jpg THAT'S ME

Also, could you add a comma between "Martin Ouellet" and "Marc Sanfaçon" (i.e., `Martin Ouellet, and Marc Sanfaçon`) 🙈

Also, HR would probably like if we could add this small line at the end:
`_Do you like leaving or receiving constructive code reviews? Do you long to work on a service where your code can improve the code of others? Check out our [careers](https://www.coveo.com/en/company/careers) page and join the team!_`
(feel free to rephrase that better if you think of something better 😛 )

**Review by amoreauCoveo — APPROVED** [review]
😎

File: _posts/2022-02-05-leveling-code-review.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Your application will be deployed, so what happens while it is still in transition? If it has to be rolled back, what happens then? Other services will interact with it – is it problematic? You are also to think in terms of organization; is this the right place to put this feature or is there another service that would be better suited to do it? You should be looking at your service as a part of the system, not as “the” system. Although all of those concerns should normally be identified during a design review, often enough some small changes might have unsuspected ramifications with deep architectural and responsibility implications. 
+
+- Is this the right place to add this functionality in the system ?
+- Is this an issue encountered by many other teams? Could this be generalized to be used by other services ?
+- What will be the impact of this change during the deployment ?
+- Is rollback possible if something goes wrong?
+- Is this properly monitorable ?
+- What is the impact of this change if more than one service is doing it at the same time
+- Is this an API breaking change ? Is this a feature breaking change?
```
Comment (sbeginCoveo) [inline _posts/2022-02-05-leveling-code-review.md]: inconsistent use of space before the interrogation point :robot: (also a few other places)

**Review by sbeginCoveo — APPROVED** [review]
Well done! 
I think it will fits very well in the tech blog. Its also refreshing, since most posts subjects are "precise" or "narrow" and this one will be relevant for almost everyone in R&D :tada:

File: _posts/2022-02-01-leveling-code-review.md [hunk 3]
```diff
@@ -64,14 +64,14 @@ At this level of code review, you should not only be taking in consideration you
...
-- Is this the right place to add this functionality in the system ?
-- Is this an issue encountered by many other teams? Could this be generalized to be used by other services ?
-- What will be the impact of this change during the deployment ?
+- Is this the right place to add this functionality in the system?
+- Is this an issue encountered by many other teams? Could this be generalized to be used by other services?
+- What will be the impact of this change during the deployment?
 - Is rollback possible if something goes wrong?
-- Is this properly monitorable ?
+- Is this properly monitorable?
 - What is the impact of this change if more than one service is doing it at the same time
```
Comment (sbeginCoveo) [inline _posts/2022-02-01-leveling-code-review.md:72]: missing ?

**Pull request merged**